config/defaults.hjson

```diff
@@ -110,4 +110,7 @@
   # Sets a response Access-Control-Allow-Origin CORS header
   # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin
   cors_origin: "lemmy.tld"
+  # If this is true, users will never see the dialog asking to support Lemmy development with
+  # donations.
+  disable_donation_dialog: true
```

Comment:
This needs to be moved to a boolean column on `local_site` like most of the other settings. Then it can be turned on and off easily, and front ends can view this setting.

Reply:
Alright moved it to the db.